ddl/column.go

```diff
@@ -932,18 +934,23 @@ func newUpdateColumnWorker(sessCtx sessionctx.Context, worker *worker, id int, t
 		backfillWorker: newBackfillWorker(sessCtx, worker, id, t),
 		oldColInfo:     oldCol,
 		newColInfo:     newCol,
+		metricCounter:  metrics.BackfillTotalCounter.WithLabelValues("update_col_speed"),
```

Comment:
"update_col_count"?

Reply:
It's the original code.

Reply:
I think if we want to update it in other PR, at the same time, we can update `BackfillTotalCounter`